sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py

```diff
@@ -354,21 +358,26 @@ def import_certificate(self, certificate_name, certificate_bytes, **kwargs):
         """
 
         enabled = kwargs.pop("enabled", None)
-        password = kwargs.pop("password", None)
         policy = kwargs.pop("policy", None)
 
         if enabled is not None:
             attributes = self._models.CertificateAttributes(enabled=enabled)
         else:
             attributes = None
         base64_encoded_certificate = base64.b64encode(certificate_bytes).decode("utf-8")
-        bundle = self._client.import_certificate(
-            vault_base_url=self.vault_url,
-            certificate_name=certificate_name,
+
+        parameters = self._models.CertificateImportParameters(
             base64_encoded_certificate=base64_encoded_certificate,
-            password=password,
+            password=kwargs.pop("password", None),
             certificate_policy=policy._to_certificate_policy_bundle(),
```

Comment:
Unrelated to this PR: I see `policy` is optional but when it isn't provided, this line raises. Would the service use some default policy if we passed `None` here?

Reply:
great catch, thanks! I'll make another PR for this fix

I checked with a service call, and it does seem like the service returns a default policy in the response body if one is not given to it

With this request body:
{"value": huge base 64 value, "pwd": "123"}'

I got this policy in the response body:
```
"policy":{
      "id":"https://vaultname.vault.azure.net/certificates/cert4f5650dfd/policy",
      "key_props":{
         "exportable":true,
         "kty":"RSA",
         "key_size":2048,
         "reuse_key":false
      },
      "secret_props":{
         "contentType":"application/x-pkcs12"
      },
      "x509_props":{
         "subject":"CN=KeyVaultTest",
         "ekus":[

         ],
         "key_usage":[

         ],
         "validity_months":297,
         "basic_constraints":{
            "ca":false
         }
      },
      "lifetime_actions":[
         {
            "trigger":{
               "lifetime_percentage":80
            },
            "action":{
               "action_type":"EmailContacts"
            }
         }
      ],
      "issuer":{
         "name":"Unknown"
      },
      "attributes":{
         "enabled":true,
         "created":1593544875,
         "updated":1593544875
      }
   }
```